cosmos/profiles/base.py

```diff
@@ -41,11 +102,13 @@ class BaseProfileMapping(ABC):
 
     _conn: Connection | None = None
 
-    def __init__(self, conn_id: str, profile_args: dict[str, Any] | None = None, disable_event_tracking: bool = False):
+    def __init__(
+        self, conn_id: str, profile_args: dict[str, Any] | None = None, dbt_config_vars: DbtConfigVars | None = None
+    ):
```

Comment:
This is a breaking change since `disable_event_tracking` was [released in v1.3.0](https://github.com/astronomer/astronomer-cosmos/releases/tag/astronomer-cosmos-v1.3.0) and cosmos follows [semver versioning](https://semver.org/), and we cannot remove `disable_event_tracking` until cosmos v2.0. So it can't be removed, but we could consider warning users in v1.4.0 (when this feature is released) that `disable_event_tracking` is deprecated and will be removed in 2.0.

Reply:
returned)